### ISSUE TITLE
Store meta data with ONNX file

### ISSUE THREAD

**Issue by carsten-wenderdel** [carsten-wenderdel/wildbg#35]
The neural nets should be reproducible. For that it would be helpful to store some [metadata](https://onnx.ai/onnx/intro/concepts.html#metadata) in the ONNX files:
- Which rollout files have been used (maybe file hash and first position?)
- Loss function
- Optimizer like `AdamW` including learning rate etc.
- Number of epochs
